docs/dependency-updates.md

```diff
@@ -49,11 +49,11 @@ Below is a list of dependencies that are locked down due to known issues with se
 
 #### axios
 
-`axios` has a major update available to ^1.0.0 but there are breaking changes and no associated upgrade guide as of writing this. See [this issue](https://github.com/axios/axios/issues/5014). Seems like as of now, there are still quite a few issues popping up with this major update so it may be worthwhile to wait a few days until those are ironed out.
+`axios` has a major update available to ^1.0.0 but there are breaking changes and no associated upgrade guide as of 10/26/2022. See [this issue](https://github.com/axios/axios/issues/5014). Seems like as of now, there are still quite a few issues popping up with this major update so it may be worthwhile to wait unil they are ironed out and/or the upgrade guide is published.
 
 #### puppeteer / puppeteer-core
 
-`puppeteer` and `puppeteer-core` have a major version update to ^18.x.x, but they need to stay at the same major version as `chrome-aws-lambda` (17.1.3). If we upgrade `puppeteer`, we see a ` cannot read property 'prototype' of undefined` error. 
+`puppeteer` and `puppeteer-core` have a major version update to ^1.x.x, but they need to stay at the same major version as `chrome-aws-lambda` (17.1.3). If we upgrade `puppeteer`, we see a ` cannot read property 'prototype' of undefined` error. 
```

Comment:
Smallest nit: I think the version should be `^19.x.x` instead of `^1.x.x`?

Reply:
Thank you! Fixed it!